### PR TITLE
[TITS-5262] Properly color boilerplate links

### PR DESCRIPTION
**Pull request by kudlajz**

### CODE REVIEW

**Review by oleg-semyonov — COMMENTED** [review]
The original issue was about headings not having the correct color (h2, h3 tags), but it's nice that you fixed links too :)

Please add a fix for headings and it will be perfect 👍🏻

**Comment by kudlajz** [comment]
Oh yeah, I was wondering why the title says "headings" 😄

File: modules/Layout/Boilerplate/Boilerplate.module.scss [hunk 1]
```diff
@@ -40,13 +40,18 @@
 
 .about {
     margin-bottom: $spacing-5;
+
+    * {
```
Comment (kudlajz) [inline modules/Layout/Boilerplate/Boilerplate.module.scss:44]: There's a bunch of options (headings 1 to 6, quote etc.) so this felt like the simplest solution that does not affect anything else.

Comment (oleg-semyonov) [reply modules/Layout/Boilerplate/Boilerplate.module.scss:44]: Should be fine, but I would go with `color: inherit`, just so we don't need to maintain it in multiple places.

**Pull request merged**